src/sagemaker_tensorflow_container/training.py

```diff
@@ -138,7 +142,11 @@ def train(env):
 
         if not _is_host_master(env.hosts, env.current_host):
             _wait_until_master_is_down(env.hosts[0])
-
+    elif mpi_enabled:
```

Comment:
Just send the enable flag, you dont need to make a conditional here.

Reply:
done